it/mail/chrome/messenger/preferences/changeaction.dtd

```diff
@@ -1,23 +1,23 @@
-<!ENTITY  changeAction.title                  "Change Action">
+<!ENTITY  changeAction.title                  "Cambia azione">
 <!ENTITY  window.width                         "36em">
 
-<!ENTITY  whenDownloading.label               "When downloading files like this:">
+<!ENTITY  whenDownloading.label               "quando scarichi file come questo:">
```

Comment:
CORRECTIONS NEEDED. A capital letter is required at the beginning of a sentence. The correct entry should be: 

 +<!ENTITY  whenDownloading.label               "Quando scarichi file come questo:">